test/unittests/mongoBackend/mongoUpdateContextSubscription_test.cpp

```diff
@@ -234,7 +234,7 @@ static void prepareDatabase(void) {
 
     BSONObj sub1 = BSON("_id" << OID("51307b66f481db11bf860001") <<
                         "expiration" << 10000000 <<
-                        "lastNotification" << 15000000 <<
+                        "lastNotification" << (long long) 15000000 <<
```

Comment:
Let's check if the final "read long or int, write long" logic works" by removing `(long long)` in all the prepareDatabase() places.

(Note that other modifications to the unit test, in particular `sub.getField("lastNotification").Long()` would be ok, in the case of fields re-writen during the test, as our write policy is to use long).